src/plugins/spdk/spdk_env/threads.c

```
@@ -0,0 +1,517 @@
+{
+	uint32_t i;
+
+	for (i = 0; i < g_core_count; i++) {
+		if (g_workers[i].running) {
+			pthread_join(g_workers[i].thread, NULL);
+			g_workers[i].running = false;
+		}
+	}
+}
+
+void spdk_unaffinitize_thread(void)
```

Cppcheck (2.21.0)

[style] The function 'spdk_unaffinitize_thread' should have static linkage since it is not used outside of its translation unit.

(staticFunction)